Formula/glibc.rb

```diff
@@ -150,13 +154,55 @@ def install
       prefix.install_symlink "lib" => "lib64"
     end
 
+    # Add ld.so.conf (will be linked to HOMEBREW_PREFIX/etc/ld.so.conf)
```

Comment:
```suggestion
    # Add ld.so.conf (will be written to HOMEBREW_PREFIX/etc/ld.so.conf)
```
`etc` is not in `prefix`.